docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -127,6 +127,16 @@ export GITEA_WORK_DIR=/var/lib/gitea/
 cp gitea /usr/local/bin/gitea
 ```
 
+### Adding bash/zsh autocompletion (from 1.19)
+
+A script to enable bash-completion can be found at `contrib/autocompletion/bash_autocomplete`. This can be copied to `/usr/share/bash-completion/completions/gitea`
```

Comment:
```suggestion
A script to enable bash-completion can be found at [`contrib/autocompletion/bash_autocomplete`](https://raw.githubusercontent.com/go-gitea/gitea/main/contrib/autocompletion/bash_autocomplete). This can be copied to `/usr/share/bash-completion/completions/gitea`
```